### ISSUE TITLE
Orchestrator colour for Data Sets

### ISSUE THREAD

**Issue by AngelaFaulding** [NHSDigital/DataDictionaryPublication#616]
@jamesrwelch - as discussed, the colour of the data set links have been changed in the live dictionary as the orange failed the accessibility check.

They are now grey, which is a colour in  the NHS Colour Palette

![image](https://github.com/user-attachments/assets/3121033e-b69e-4902-96fe-03d22539b729)

The Orchestrator still has Orange assigned to data sets:
 
![image](https://github.com/user-attachments/assets/b7d9c1a7-9a77-4f7c-bc3d-64eae31628b9)

Can data sets be changed in the Orchestrator to #425563 please?

**Comment by AngelaFaulding** [comment]
Data Sets now have the correct grey colour in the Orchestrator:

![Image](https://github.com/user-attachments/assets/ce46b107-c9f7-4578-93ea-af8b74145655)
